packages/node_modules/@webex/plugin-meetings/src/metrics/index.js

```diff
@@ -161,6 +161,7 @@ class Metrics {
       originTime: {
         triggered: new Date().toISOString()
       },
+      senderCountryCode: this.webex.meetings.geoHintInfo?.countryCode,
```

Comment:
Should this be optional property or not sending undefined if it doesn't exist?

Reply:
I tested it out. If `getHintInfo` is undefined, the field does not exist in the payload to the server.